agr_literature_service/api/crud/utils/yml/resource.yml

```diff
@@ -0,0 +1,39 @@
+#
+# This is pretty much the same as the reference apart from the ZFIN one.
+# It might be less duplication in the yml to copy the reference dict in the api
+# and then just make changes based on this file and just have difference ones here
```

Comment:
I think only ZFIN and FB have resources, so it might be confusing if people see this and think it means there are resources for other mods ?  Might want some feedback from other developers, or even curators about future resource plans

Reply:
Okay that is easy, I can delete the other lines that are not needed. Thanks.